packages/@okta/vuepress-site/docs/reference/api/users/index.md

```diff
@@ -3531,6 +3641,8 @@ Metadata properties such as `id`, `status`, timestamps, `_links`, and `_embedded
 * The `activated` timestamp will only be available for users activated after 06/30/2013.
 * The`statusChanged` and `lastLogin` timestamps will be missing for users created before 06/30/2013 and updated on next status change or login.
 
+The `type` property is a map that identifies the User Type of the user (see <ApiLifecycle access="ea" /> [User Types](/docs/reference/api/user-types)). Currently it contains a single element, `id`, as shown in the Example. It can be specified when creating a new user, but once the user is created the value is read-only.
```

Comment:
When referring to the User object, you should capitalize "User".

Reply:
It's a fuzzy line between referring to the User object and referring to the user person.  I'd guess the two appearances of "user" in the last sentence must refer to the object, since we don't "create" the person, but the mention in the first sentence (User Type of the user) is less clear.  I'll update the two later occurrences but am leaving that earlier one lower-case unless you push back.

On a different note, I see here I included the lifecycle tag ahead of the `[User Types]` link, but I'm sure I have not been consistent about including it in other parts of this PR.  I'll make a pass and touch those up.